lib/improver/tests/wxcode/wxcode/test_wxcode_utilities.py

```diff
@@ -93,6 +96,21 @@ def setUp(self):
                          2, 0, 1, 29, 30, 1, 5, 6, 6]).reshape(2, 1, 3, 3)
         self.cube = set_up_cube(data, 'air_temperature', 'K',
                                 realizations=np.array([0, 1]))
+        self.wxcode = np.array(sorted(WX_DICT.keys()))
+        self.wxmeaning = ''
```

Comment:
I think this can be simplified to:
```python
wxstring = " ".join(WXDICT.values())
```

Reply:
This seems to work (although it is WX_DICT.values()) however the order is very important. Can we be sure the dictionary will always return the values in the right order?

Reply:
If you're concerned about ordering, then it might be best to define `WX_DICT` as an OrderedDict (https://docs.python.org/2/library/collections.html#collections.OrderedDict) and then the order can be guaranteed. OrderedDicts are already used within the repository.